### ISSUE TITLE
Unrecognized option "default_section" under "presta_sitemap".

### ISSUE THREAD

**Issue by crabnky** [prestaconcept/PrestaSitemapBundle#199]
Hi, I'm trying to configure the default section. According to docs I should add "default_section" under "presta_sitemap" in presta_sitemap.yaml. But then I see the following:
```
Unrecognized option "default_section" under "presta_sitemap". Available options are "defaults", "dumper", "generator", "items_by_set", "route_annotation_listener", "sitemap_file_prefix", "timetolive".
````

Did I miss something?
Thanks

**Comment by yann-eugone** [comment]
Hi @crabnky you are reading `master` documentation, and using a feature that is not tagged yet.

Please read documentation for the version you are actually using : 

* [1.6](https://github.com/prestaconcept/PrestaSitemapBundle/blob/v1.6.0/Resources/doc/2-configuration.md)
* [1.5.4](https://github.com/prestaconcept/PrestaSitemapBundle/blob/v1.5.4/Resources/doc/2-configuration.md)

**Comment by crabnky** [comment]
Ah I see, all fine then. Thanks! 👍

**Comment by yann-eugone** [comment]
For the record

Released in [1.7.0](https://github.com/prestaconcept/PrestaSitemapBundle/releases/tag/v1.7.0)
